### ISSUE TITLE
Project does not compile

### ISSUE THREAD

**Issue by dobri-002** [KDAB/integrating-qq2-with-opengl#1]
I receive the following error when I try to compile the project:

```
error: dependent 'integrating-qq2-with-opengl\underlay\..\lib\librenderlib.a' does not exist.
```

My environment:
Windows 7 / Windows 10
Qt 5.5.1
Qt Creator 3.5.1 (opensource), Based on Qt 5.5.1 (MSVC 2013, 32 bit)
compiler: MSVC 2013 64bit

**Comment by dangelog** [comment]
Yes, I've only tested it under Linux. I should fix the .pro to work under Windows as well.

**Comment by xarthurx** [comment]
Does this issue got fixed?
